torch/_subclasses/fake_tensor.py

```diff
@@ -581,10 +581,12 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
                 if symbolic_shapes.is_symbolic_op(func):
                     return symbolic_shapes.handle_symbolic_op(func, args, kwargs)
                 if func == aten.size.default:
-                    raise RuntimeError(
+                    print(
                         "Trying to call aten.size on a tensor with symbolic shapes. "
                         "It's likely that this is from calling tensor.shape in C++"
                     )
+                    # We do this to allow for better error localization with `TORCH_SHOW_CPP_STACKTRACES=1`
```

Comment:
why does not raising here make it easier to find a stacktrace? does `return None` trigger someone else to raise, and their stacktrace is more useful?

Reply:
A better fix for this is to localize Python errors that go back into C++. But at minimum, the print here should go to sys.stderr